### ISSUE TITLE
Create "Grants Withdrawal" Page - Design

### ISSUE THREAD

**Issue by captnseagraves** [gitcoinco/web#1907]
### User Story

As a Grant creator, I want to withdraw funds that have been given to the project on a monthly basis so that I can sustain myself and my team

### Why Is this Needed

The "Grants Withdrawal" page is needed to allow a grant creator to withdrawal funds that have been given to the project

### Description

*Type*: Feature

### Definition of Done

- [ ] Page displays the total funding the grant has received
- [ ] Page displays the current withdrawable amount
- [ ] Page has a "Withdraw Funds" button
- [ ] Page displays a Withdrawal History of previous withdrawal activity in the grant
- [ ] Page has standard Gitcoin header and footer
- [ ]  Design must be on brand, see creative repo for brand guide

### Additional Information

To be reviewed with @PixelantDesign @mbeacom @captnseagraves

Existing assets can be found at this PR: https://github.com/gitcoinco/web/pull/674
More information about the Dev Grants project can be found here: https://github.com/gitcoinco/web/issues/1469

Please allow the core team a few rounds of revisions.

**Comment by captnseagraves** [comment]
Assigning @justpixel

**Comment by justpixel** [comment]
Do you think a modal window is enough? Or it should have a separate page?
![withdraw funds](https://user-images.githubusercontent.com/7627020/43709811-7ef0c580-9976-11e8-804f-e0f8bd207a30.png)

**Comment by owocki** [comment]
i think we decided this should be a page not a modal right? 

other than that LGTM

**Comment by PixelantDesign** [comment]
yes looks good! let's put on a page. 

What happens after I click withdraw? Metamask?

**Comment by justpixel** [comment]
![withdraw funds](https://user-images.githubusercontent.com/7627020/44090028-748a4eca-9fd1-11e8-90e0-2ad0869d4324.png)

**Comment by justpixel** [comment]
Design:
![withdraw funds](https://user-images.githubusercontent.com/7627020/44267234-25264980-a236-11e8-987a-cc19187d35fc.png)

**Comment by captnseagraves** [comment]
Yes, I think having USD displayed would be nice! Could you add that to the design @justpixel ?
